charts/publisher/templates/web/dbmigration-job.yaml

```diff
@@ -1,21 +1,23 @@
-{{- if .Values.dbMigrationEnabled -}}
+{{- if .Values.web.dbMigrationEnabled -}}
 apiVersion: batch/v1
 kind: Job
 metadata:
   name: {{ .Release.Name }}-dbmigrate-{{ now | unixEpoch }}
+  labels:
+    {{- include "publisher.labels" . | nindent 4 }}
+    app.kubernetes.io/component: "web"
 spec:
   template:
     metadata:
-      name: "{{ .Release.Name }}"
+      name: {{ .Release.Name }}-dbmigrate-{{ now | unixEpoch }}
```

Comment:
Optional: might be marginally more user-friendly to use a human-readable timestamp, e.g. yyyymmddhhmmss (i.e. `now | date "20060102150405"`)